discord/dockerfiles/hs-Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM haskell:9-buster
```

Comment:
Is there an advantage to using `buster` over `slim-buster`?

Reply:
No, that was my mistake; for this use case it should probably be `9-slim-buster`. I'll go ahead and switch it over.